src/setup/keyboard.c

```diff
@@ -230,7 +231,7 @@ static void ConfigExtraKeys(TXT_UNCAST_ARG(widget), TXT_UNCAST_ARG(unused))
         AddKeyControl(table, "Toggle vert. mouse", &key_togglenovert);
         AddKeyControl(table, "Quick Reverse", &key_reverse);
         }
-        else if (gamemission == heretic || gamemission == hexen)
+        else if (gamemission == heretic || gamemission == hexen || gamemission == strife)
```

Comment:
So, this boils down to plain `else` here.

Reply:
Fixed by https://github.com/fabiangreffrath/crispy-doom/pull/998/commits/90f112349863ab2944fbb8cfaa4d749daf9c6294.